apps/etterlatte-api/src/main/kotlin/no/nav/etterlatte/samordning/sak/BehandlingSakRoutes.kt

```diff
@@ -1,16 +1,20 @@
 package no.nav.etterlatte.samordning.sak
```

Comment:
Bør kanskje namespace oppdateres for disse tingene også? Det er vel ikke kun forbeholdt samordning lenger

Reply:
Ja, har tenkt på det, men vil ta navnendring/omstrukturering i egen PR for å redusere omfanget.